app/layout.tsx

```diff
@@ -0,0 +1,14 @@
+import "../styles/globals.css";
+
+type Props = {
+  children: React.ReactNode;
```

Comment:
여기서도 이렇게 표현되는게 좋을 것 같아요~
```suggestion
  children: ReactNode;
```